src/Swarm/Game/Scenario.hs

```diff
@@ -92,7 +101,8 @@ data Scenario = Scenario
   , _scenarioEntities :: EntityMap
   , _scenarioRecipes :: [Recipe Entity]
   , _scenarioKnown :: [Text]
-  , _scenarioWorld :: WorldDescription
+  , _scenarioWorlds :: NonEmpty WorldDescription
```

Comment:
If the root world must always come first, let's document that invariant somewhere.